app/assets/javascripts/initializers/initNotifications.js

```diff
@@ -1,3 +1,5 @@
+/* global checkUserLoggedIn, instantClick, InstantClick, sendHapticMessage */
```

Comment:
Aside from this eslint comment, the remainder of the changes in this file, unless noted, are all prettier code formatting.